adapter/adapter.go

```diff
@@ -86,6 +86,8 @@ var (
 
 // Adapter represents a connection to a chat provider.
 type Adapter interface {
+	BookmarkLink(ctx context.Context, channelID, title, link string) error
```

Comment:
Could you elaborate on the purpose and use cases for this function in a comment?